tun/Cargo.toml

```diff
@@ -14,6 +14,7 @@ tracing = "0.1"
 log = "0.4"
 serde = { version = "1", features = ["derive"], optional = true }
 schemars = { version = "0.8", optional = true }
+rtnetlink = "0.14"
```

Comment:
`rtnetlink` should go in a section like this `[target.'cfg(target_os = "linux")'.dependencies]` to fix the macOS build error